src/libs/LangChain.Core/Chains/StackableChains/Agents/Crew/AgentTask.cs

```diff
@@ -1,48 +1,22 @@
-﻿using LangChain.Chains.StackableChains.Agents.Crew.Tools;
+﻿using LangChain.Abstractions.Chains.Base;
+using LangChain.Chains.StackableChains.Agents.Crew.Tools;
+using LangChain.Chains.StackableChains.ReAct;
 
 namespace LangChain.Chains.StackableChains.Agents.Crew;
 
-/// <summary>
-/// 
-/// </summary>
-/// <param name="agent"></param>
-/// <param name="description"></param>
-/// <param name="tools"></param>
-public class AgentTask(
-    CrewAgent agent,
-    string description,
-    List<CrewAgentTool>? tools = null)
+public class AgentTask(CrewAgent agent, string description, List<CrewAgentTool>? tools=null)
 {
-    /// <summary>
-    /// 
-    /// </summary>
     public CrewAgent Agent { get; set; } = agent;
-    
-    /// <summary>
-    /// 
-    /// </summary>
-    public List<CrewAgentTool> Tools { get; set; } = tools ?? new List<CrewAgentTool>();
-    
-    /// <summary>
-    /// 
-    /// </summary>
+    public List<CrewAgentTool> Tools { get; set; } = tools??new List<CrewAgentTool>();
     public string Description { get; set; } = description;
 
-    /// <summary>
-    /// 
-    /// </summary>
-    /// <param name="context"></param>
-    /// <param name="cancellationToken"></param>
-    /// <returns></returns>
-    public async Task<string> ExecuteAsync(
-        string? context = null,
-        CancellationToken cancellationToken = default)
+    public string Execute(string context=null)
     {
         Agent.AddTools(Tools);
         Agent.Context = context;
         var chain = Chain.Set(Description, "task") 
                     | Agent;
-        var res = await chain.Run("result").ConfigureAwait(false) ?? string.Empty;
+        var res = chain.Run("result").Result;
         return res;
```

Comment:
Don't use .Result for async tasks. It can create problems for desktop apps.
Need to refactor method to be Async